Cargo.toml

```diff
@@ -66,6 +66,9 @@ pprof = { version = "0.9", features = ["criterion", "flamegraph"] } # MSRV 1.56
 bench = false
 
 [features]
+default = ["halo2-batch", "halo2-multicore"]
```

Comment:
My concern here is that these feature flags don't actually change anything in `orchard` code; they're there exclusively because Rust doesn't allow you to disable default features of a transitive dependency in any other way. So they really are about halo2, not orchard.

Reply:
If `orchard` did not require `multicore` for performance, then we would just leave it disabled here completely, and then leave it up to the end user to enable `multicore` on `halo2_proofs` if they so choose. But given that we know the default use case for `orchard` requires it for performance, we need it on by default, and thus exposed as `orchard` functionality.

Reply:
Put another way, this is effectively the same as having a `std` flag. If a crate doesn't itself use `std` features, it can be `no-std` compatible by annotating itself as such and disabling any `std` feature flags on its dependencies. But if it _does_ require `std` for its default use case, then it offers a `std` feature flag to allow downstreams to selectively disable it. We don't prefix `std` based on which of our dependencies requires it by default.

Reply:
Fixed in 16a143c3c70b93c1583b7ecd3d8bea70cfadbfa0